torchcrf/__init__.py

```diff
@@ -151,16 +153,18 @@ def _validate(
                 f'got {emissions.size(2)}')
 
         if tags is not None:
-            if emissions.shape[:2] != tags.shape:
+            if emissions.shape[0] != tags.shape[0] or emissions.shape[1] != tags.shape[1]:
```

Comment:
Why is this changed? Same question for other similar changes below.

Reply:
This line of change is not necessary, but the cast to tuple below will error.